settings/provisioner/virsh/virsh.spec

```diff
@@ -19,13 +19,9 @@ subparsers:
                       default: ~/.ssh/id_rsa
             - title: image
               options:
-                  image-file:
-                      type: Value
-                      help: An image to provision the host with
-                      required: yes
-                  image-server:
-                      type: Value
-                      help: Base URL of the image file server
+                  image:
+                      type: YamlFile
+                      help: The RHEL distribution version
```

Comment:
The help text is wrong here. please explain that this is the path to a file containing the image and the server details